os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -1257,6 +1257,22 @@ static void app_driver_call_os_func_init(void)
 	driver_call_os_func_map.driver_exit_critical = vPortExitCritical;
 }
 
+IMAGE2_RAM_TEXT_SECTION
+void app_RAM_data_copy(void)
```

Comment:
Is it verified? The data copy is verified at run time?

Reply:
ya, i have verify it.
It can copy RAM data during app_start. Data in RAM section will be reset.